src/Evaluate.jl

```diff
@@ -79,20 +80,33 @@ function eval_tree_array(
         _is_loopvectorization_loaded(0) ||
             error("Please load the LoopVectorization.jl package to use this feature.")
     end
+    if (v_turbo isa Val{true} || v_turbo isa Val{true}) && !(T <: Number)
+        error("Bumper feature only works with numbers")
+    end
     if v_bumper isa Val{true}
         return bumper_eval_tree_array(tree, cX, operators, v_turbo)
     end
 
     result = _eval_tree_array(tree, cX, operators, v_turbo)
-    return (result.x, result.ok && !is_bad_array(result.x))
+    return (result.x, result.ok && is_valid_array(result.x))
+end
+
+function eval_tree_array(
+    tree::AbstractExpressionNode{T},
+    cX::AbstractVector{T},
+    operators::OperatorEnum;
+    kws...
+) where {T}
+    return eval_tree_array(tree, reshape(cX, (size(cX)[1], 1))::AbstractMatrix{T}, operators; kws...)
```

Comment:
```suggestion
    return eval_tree_array(tree, reshape(cX, (size(cX, 1), 1)), operators; kws...)
```

Although I wonder if this might create some issues if someone passes in a vector of vectors (do we want to allow that?)